pytest_pylint.py

```diff
@@ -81,9 +81,14 @@ def pytest_collect_file(path, parent):
             ignore_list = ignore_string.split(',')
     except (NoSectionError, NoOptionError):
         pass
+    msg_template = None
+    try:
+        msg_template = pylintrc.get('REPORTS', 'msg-template')
+    except (NoSectionError, NoOptionError):
```

Comment:
Since this will raise the same exceptions as the try block above and we aren't doing anything with those exceptions, I think it would be cleaner to move that up into that block.